sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/SocketsCreation.vb

```diff
@@ -32,7 +32,7 @@ Namespace Tests.TestCases
     Public Class MySocket
         Inherits Socket
         Sub New()
-            MyBase.New(Nothing)
+            MyBase.New(CType(Nothing, SocketInformation))
```

Comment:
Constructor overload resolution fails under .Net 5 since a new constructor was added and calling the base with Nothing will match more than one.

https://docs.microsoft.com/en-us/dotnet/api/system.net.sockets.socket.-ctor?view=net-5.0#overloads